Dockerfile.base

```diff
@@ -15,37 +15,9 @@ RUN apt-get update && \
     apt-get install -y python3-dev python3-pip python3-numpy python3-scipy \
         python3-matplotlib
 
-###############################################################################
-# Install OpenCV 3.4.3 with CUDA support (take a seat, this takes hours)
```

Comment:
where is opencv, now? is it installed by tensorflow?

Reply:
It is now installed by deeplabcut as a part of the package